packages/redis-v5/commands/cli.js

```diff
@@ -142,6 +142,10 @@ function maybeTunnel (redis, config) {
   let uri = url.parse(redis.resource_url)
   let prefer_native_tls = redis.prefer_native_tls
 
+  if (prefer_native_tls && hobby) {
+    uri = url.parse(match(config, /_TLS_URL/))
```

Comment:
what if they have multiple redis hobbies attached? Will the config have multiple `_TLS_URL` in the config? I'm not sure if the config is just for the single add-on or the entire app.

Reply:
I think I answered my question. Looks like it only retrieves the add-on's config vars, not the full apps

Reply:
I tested this in dev and it works with multiple hobby-dev redis on an app. It gets the config var for the addon ( see `getRedisConfigVars` on L212)

Reply:
excellent